2025-06-30 09:02:54,830 - __main__ - ERROR - initialize_services:143 - Error initializing services: Client.__init__() got an unexpected keyword argument 'proxies'
2025-06-30 09:04:23,650 - models.conversation - ERROR - _create_indexes:47 - Error creating indexes: SSL handshake failed: ac-gq5sqvd-shard-00-00.j3gddli.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:992),SSL handshake failed: ac-gq5sqvd-shard-00-02.j3gddli.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:992),SSL handshake failed: ac-gq5sqvd-shard-00-01.j3gddli.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:992), Timeout: 30s, Topology Description: <TopologyDescription id: 68628b391ddcb7cc7d5e8ebd, topology_type: ReplicaSetNoPrimary, servers: [<ServerDescription ('ac-gq5sqvd-shard-00-00.j3gddli.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('SSL handshake failed: ac-gq5sqvd-shard-00-00.j3gddli.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:992)')>, <ServerDescription ('ac-gq5sqvd-shard-00-01.j3gddli.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('SSL handshake failed: ac-gq5sqvd-shard-00-01.j3gddli.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:992)')>, <ServerDescription ('ac-gq5sqvd-shard-00-02.j3gddli.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('SSL handshake failed: ac-gq5sqvd-shard-00-02.j3gddli.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:992)')>]>
